### PR TITLE
TizenRenderer: Remove renderer type getter

### PR DESCRIPTION
**Pull request by JSUYA**
The embedder does not require a type getter of renderer because -rtti option is enabled.
So, remove getter method.

### CODE REVIEW

File: flutter/shell/platform/tizen/tizen_renderer.h [hunk 1]
```diff
@@ -26,8 +26,6 @@ class TizenRenderer {
 
   bool IsValid() { return is_valid_; }
 
-  FlutterDesktopRendererType type() { return type_; }
-
```
Comment (swift-kim) [inline flutter/shell/platform/tizen/tizen_renderer.h:30]: You can remove this unnecessary include:
```cpp
#include "flutter/shell/platform/tizen/public/flutter_tizen.h"
```

File: flutter/shell/platform/tizen/flutter_tizen.cc [hunk 2]
```diff
@@ -284,8 +285,8 @@ void FlutterDesktopViewOnKeyEvent(FlutterDesktopViewRef view,
       ViewFromHandle(view)->tizen_view());
 
   if (tizen_view->GetType() == flutter::TizenViewType::kView &&
-      ViewFromHandle(view)->engine()->renderer()->type() ==
-          FlutterDesktopRendererType::kEGL) {
+      dynamic_cast<flutter::TizenRendererEgl*>(
+          ViewFromHandle(view)->engine()->renderer())) {
     reinterpret_cast<flutter::TizenViewNui*>(tizen_view)
```
[the remark this answers is not in the document]
Comment (JSUYA) [reply flutter/shell/platform/tizen/flutter_tizen.cc]: I updated code. thanks.

File: flutter/shell/platform/tizen/tizen_renderer.h (hunk 1 above)
Comment (JSUYA) [reply flutter/shell/platform/tizen/tizen_renderer.h:30]: done

File: flutter/shell/platform/tizen/flutter_tizen_engine.cc [hunk 3]
```diff
@@ -160,7 +160,7 @@ bool FlutterTizenEngine::RunEngine() {
 
   FlutterTaskRunnerDescription render_task_runner = {};
 
-  if (IsHeaded() && renderer_->type() == FlutterDesktopRendererType::kEvasGL) {
+  if (IsHeaded() && dynamic_cast<TizenRendererEvasGL*>(renderer_.get())) {
```
[the remark this answers is not in the document]
Comment (JSUYA) [reply flutter/shell/platform/tizen/flutter_tizen_engine.cc:163]: I left my comments in that comment thread. Please check

**Review by swift-kim — APPROVED** [review]

**Comment by JSUYA** [comment]
resolved conflict

**Review by bbrto21 — APPROVED** [review]

**Pull request merged**